modules/azure-landing-zone/frontdoor.tf

```diff
@@ -151,6 +151,7 @@ resource "azurerm_frontdoor" "main" {
       forwarding_configuration {
         forwarding_protocol                   = "HttpOnly"
         backend_pool_name                     = host.value["name"]
+        cache_enabled                         = lookup(host.value, "cache_enabled", "true")
```

Comment:
should this be boolean `true`? 

Can you just run this against sbox either locally or by commenting part of the condition to make sure it works

Reply:
The plan looks ok, doesn't that seem fine ?